src/Examples/UnitModels/cstr_testing.ipynb

```diff
@@ -0,0 +1,1123 @@
+    "                                              CSTR)\n",
+    "\n",
+    "from idaes.core.util.constants import Constants as c\n",
+    "from idaes.core.util import get_solver\n",
+    "from idaes.core.util.model_statistics import degrees_of_freedom\n",
+    "from idaes.core.util.initialization import (propagate_state,\n",
+    "                                            fix_state_vars,\n",
+    "                                            revert_state_vars)\n",
+    "\n",
+    "import idaes.logger as idaeslog"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Importing required thermo and reaction package\n",
+    "\n",
+    "The final set of imports are to import the thermo and reaction package. We have created a custom thermo package that support ideal vapor and liquid behavior for this system, and in this case we will restrict it to ideal liquid behavior only.\n",
+    "\n",
+    "The reaction package here assumes Arrhenius kinetic behavior for the CSTR, for which $k_0$ and $E_a$ are known *a priori* (if unknown, they may be obtained using one of the parameter estimation tools within IDAES).\n",
+    "\n",
+    "$ r = -kVC_{EO} $, $ k = k_0 e^{(-E_a/RT)}$, with the variables as follows:\n",
+    "\n",
+    "$r$ - reaction rate extent in moles of ethylene oxide consumed per second; note that the traditional reaction rate would be given by $rate = r/V$ in moles per $m^3$ per second  \n",
+    "$k$ - reaction rate constant per second  \n",
+    "$V$ - volume of CSTR in $m^3$, note that this is *liquid volume* and not the *total volume* of the reactor itself  \n",
+    "$C_{EO}$ - bulk concentration of ethylene oxide in moles per $m^3$ (the limiting reagent, since we assume excess catalyst and water)  \n",
+    "$k_0$ - pre-exponential Arrhenius factor per second  \n",
+    "$E_a$ - reaction activation energy in kJ per mole of ethylene oxide consumed  \n",
+    "$R$ - gas constant in J/mol-K  \n",
+    "$T$ - reactor temperature in K\n",
+    "\n",
+    "These calculations are contained within the property, reaction and unit model packages, and do not need to be entered into the flowsheet. More information on property estimation may be found below:\n",
+    "\n",
+    "ParamEst parameter estimation: https://idaes-pse.readthedocs.io/en/stable/user_guide/workflow/data_rec_parmest.html?highlight=paramest  \n",
+    "HELMET thermodynamic estimation: https://idaes-pse.readthedocs.io/en/stable/user_guide/modeling_extensions/surrogate/helmet/index.html  \n",
+    "RIPE reaction estimation: https://idaes-pse.readthedocs.io/en/stable/user_guide/modeling_extensions/surrogate/ripe/index.html  \n",
+    "\n",
+    "Let us import the following modules from the same directory as this jupyter notebook:\n",
+    "      <ul>\n",
+    "         <li>egprod_ideal as thermo_props</li>\n",
+    "         <li>egprod_reaction as reaction_props </li>\n",
+    "      </ul>\n",
+    "</div>"
```

Comment:
Not sure what this is, and it might just be GitHub, but this appeared in the rendering of the notebook.

Reply:
`</div>` is a tag to end the section, the lines above it create a bulleted list (it renders correctly if you view the file). I'll change it to match the bulleted list on line 62, since that syntax looks a bit cleaner.